### ISSUE TITLE
NTR: Otoacoustic emissions test abnormal (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7810]
New term request Otoacoustic emissions test abnormal (elthat_120816143749): OAE  UMLS: 1273548

**Comment by pnrobinson** [comment]
@MickeySegal

The main use of this test is in newborn screening where the more specific label would be
Absent otoacoustic emissions
Def: Failure to record otoacoustic emissions (OAE), i.e.,  sounds generated from the cochlea in response to an acoustic stimulus and transmitted across the middle ear to the external ear canal.
 PMID:35593808 

thoughts?

**Comment by MickeySegal** [comment]
I checked all our uses of this, and although some just stated that the test was abnormal, it seemed from the clinical context that the emissions were absent.  I changed the name of our finding.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000182
- label: Absent otoacoustic emissions
- parents: [Abnormal ear physiology (HP:0031704)](https://hpo.jax.org/app/browse/term/HP:0031704)
- pmid: PMID:35593808
- def: Failure to record otoacoustic emissions (OAE), i.e., sounds generated from the cochlea in response to an acoustic stimulus and transmitted across the middle ear to the external ear canal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7810
- synonyms:
